app/controllers/v0/upload_supporting_evidences_controller.rb

```diff
@@ -3,6 +3,14 @@
 module V0
   class UploadSupportingEvidencesController < ApplicationController
     include FormAttachmentCreate
+    extend ThirdPartyTransactionLogging::MethodWrapper
+
     FORM_ATTACHMENT_MODEL = SupportingEvidenceAttachment
+
+    wrap_with_logging :save_attachment_to_cloud!, additional_logs: {
+      form: '526ez supporting evidence attachment',
+      action: "upload: #{FORM_ATTACHMENT_MODEL}",
+      upstream: "S3 bucket: #{Settings.evss.s3.bucket}"
```

Comment:
It would be good to add something that can be tied back to something existing in the DB here I think if it is available easily... `InProgressForm` id for example, or `User` id. Or both. 

I think both of these are available in the controller context/scope here already, but you would need to check/confirm.

Reply:
so we have `current_user.account_uuid` in the logs.  Afaik the `User` object is just a wrapper that doesn't have an underlying DB model... 🤔 To that end, i think grabbing the `InProgressForm` id with the account_uuid value makes sense.

Reply:
I kinda feel like just the uuid should be enough, as that allows us to load a users current forms and any other associated info.  Also, having just the single data point means we don't have to make multiple trips to the DB to get info.